### DISCUSSION TITLE
What is the best practice using this module

### DISCUSSION THREAD

**Discussion by zhichen-moodys** [mrjono1/joi-to-typescript#391] [General]
Hi team, 
This module looks pretty cool, I want to use it in my project. What is the best way for generating the schema files to typescript interfaces?
Ideally, I would like it to be able to  export the schema to interface directly. Something like joi-extract-type, but that module has so many compatibility issues with hapi validator.
Converting schema files to typescript interface (DTO) files are fine, but I don't seem to be able to find an easy way to do that.
This seems like a common need, please point me to the right direction. Thanks.

Zhi

**Comment by mrjono1** [comment]
see the example folder for a way of using this project further.

Adding extract type to this project would be great, but Im not in a position to add that functionality
